cloud_governance/aws/tag_cluster/tag_cluster_resouces.py

```diff
@@ -46,7 +46,7 @@ def __append_input_tags(self, current_tags: list = None):
 
     def __check_name_in_tags(self, tags: list, resource_id: str):
         """
-        This method checks Name is present in the Tags or not, if not ot add Name tag
+        This method checks Name is present in the Tags or not, if not  add Name tag
```

Comment:
Extra space and need to check if it cg-Name

Reply:
No need to check cg-Name, we add Name,  only if Name not exist